docs/libcurl/curl_share_setopt.3

```diff
@@ -94,6 +94,11 @@ The Public Suffix List stored in the share object is made available to all
 easy handle bound to the later. Since the Public Suffix List is periodically
 refreshed, this avoids updates in too many different contexts.
 
+.IP CURL_LOCK_DATA_HOSTS_NAME
+When Transfering multiple file with SSH2, the access of the known_host file is not protected. 
+To protect the access of the file, use this option to protect the access of the file
+Be careful, only one file is authorized. Because the handle of the file is shared across connections
+
```

Comment:
The rest of this file wraps the content at around column 80, maybe you should for this section as well? And let's work on the text, I really don't understand it. My attempt:

> Share the `known_hosts` handle for SFTP and SCP transfers. Only works with the libssh2 backend. Added in 7.74.0

Reply:
Also note that you added this new section *in the midddle* of the `CURL_LOCK_DATA_PSL` explanation.

Reply:
ok corrected